### ISSUE TITLE
alt='' não é necessariamente um incumprimento de WCAG 2.1 AA

### ISSUE THREAD

**Issue by hugopeixoto** [marado/RNID#63]
De https://twitter.com/lucalanca/status/1295344667450646529

> [..] from an accessibility standpoint, alt=‘’ is valid for representing decorative images. source: https://w3.org/WAI/tutorials/images/decorative/

https://www.w3.org/WAI/WCAG21/Understanding/non-text-content:

> Situation F: If the non-text content should be ignored by assistive technology:
> 
> - Implementing or marking the non-text content so that it will be ignored by assistive technology using one of the following Techniques to indicate that text alternatives are not required for Situation F :
> 
> Techniques to indicate that text alternatives are not required for Situation F:
> - C9: Using CSS to include decorative images
> - FLASH3: Marking objects in Flash so that they can be ignored by AT
> - **H67: Using null alt text and no title attribute on img elements for images that AT should ignore**
> - PDF4: Hiding decorative images with the Artifact tag in PDF documents

O access monitor do accessibilidade.gov.pt indica a presença de alt vazio como algo a verificar manualmente: http://accessmonitor.acessibilidade.gov.pt/amp/results/siac.vet

Esta verificação está a ser utilizada nos scripts:
- scripts/05-governo.sh
-  scripts/38-siac.sh

Embora nos casos existentes haja imagens não decorativas a usar `alt=''`, acho que deviamos trocar por outra verificação para evitar que corrijam e continuemos a detectar incumprimento.

**Comment by marado** [comment]
Merged!

**Comment by marado** [comment]
Resolvido com o #68 .
